apps/api/v1/pages/api/bookings/_get.ts

```diff
@@ -186,16 +186,7 @@ function buildWhereClause(
     };
   } else {
     whereClause = {
-      OR: [
```

Comment:
Doing this `or` in Postgres for the attendees table that has millions of records is very slow.

Reply:
The execution plan shows it doing a seq scan